content/02.introduction.md

```diff
@@ -119,9 +120,68 @@ Pathogenesis is made up of three major components: entry, replication, and sprea
+Innovation in therapeutics often seeks to supplement or bolster the immune response in order to enhance the body's ability to eliminate a pathogen.
+However, disease symptoms can be caused either by a pathogen or by the immune response to the pathogen, and a host immune system's efforts to fight an infection can also sometimes cause significant damage to the host [@doi:978-0815332183; @doi:9780071283663].
+Therefore, understanding how the host responds to SARS-CoV-2 and when this response becomes dangerous is critical to the treatment of COVID-19 and to the identification and/or development of possible therapeutics.
+
+##### hCOV and the Immune Response
+
+Research in other coronaviruses that affect humans provides some indication of how SARS-CoV-2 infection proceeds in spite of the human immune response.
+While the first line of defense for the immune system is physical barriers such as the skin and mucus, viruses such as SARS-CoV that infect the epithelium are known to be adept at bypassing these defenses [@doi:10.1038/mi.2015.127]. 
+However, research in related viruses suggests that SARS-CoV-2 may possess some mechanisms that facilitate evasion of detection by monoctyes.
+Double-stranded RNA (dsRNA), which is produced during the viral replication process, is a PAMP that would typically initiate a response from the innate immune system [@doi:10.3389/fimmu.2018.00829].
+However, _in vitro_ analysis of nidoviruses including SARS-CoV suggests that these viruses can induce the development of double-membrane vesicles that protect the dsRNA signature from being detected by the host immune system [@doi:10.1128/jvi.02501-05].
+This protective envelope can therefore insulate these coronaviruses from the innate immune response's detection mechanism [@doi:10.1016/j.jpha.2020.03.001].
+Presentation of the antigen is important because this step allows it to be detected by B-lymphocytes, also called B-cells, which are a type of white blood cell with immunoglobulins bound to the membrane.
+Individual B-cells have unique membrane-bound immunoglobulins (i.e. membrane-bound antibodies).
+A B-cell is able to bind to an antigen only if its immunoglobulin is a match.
+This process is guided by proteins called pattern-recognition receptors (PRRs), which include toll-like receptors (TLRs) that respond to specific molecular signatures associated with classes of pathogens.
+For example, toll-like receptor 3 (TLR3) can detect double-stranded RNA (dsRNA) present during viral replication, allowing it to recognize a viral threat.
+When TLR7 detects single-stranded RNA, it can initiate the production of interferon, a cytokine, to stimulate nearby cells to mount an immune response to a virus [@isbn:9780321743671]. 
+The binding of a pattern recognition receptor (PRR), such as a TLR, to the antigen allows the B-cell to engulf it the virus via endocytocis.
```

Comment:
@nilswellhausen would you look at this sentence on line 168 as well? 2 standard biology textbooks I looked at only showed B-cell endocytosing antigen it recognized through B-cell receptor and then blablabla (making into smaller parts...) & then presenting with MHCII receptor to T-Helper cell. This sentence line 168 seems to say that there are also cases where you can substitute "TLR" instead of "B-cell receptor" in what I wrote in this comment, but I don't know/remember if that's true or common? Or does this sentence mean to say that TLR in some other indirect way interacts?? @rando2 maybe re-check your source? or easiest if @nilswellhausen or another person with lots of immunology experience answers (I can get an answer if needed, but just asked here because I noticed new feedback).

Reply:
Yeah I don't think that function of TLR's is to allow the B cell to engulf the virus. They recognize things like dsRNA etc and activate pro-inflammatory signaling pathways (NFkB, Irak etc).